src/App.tsx

```diff
@@ -1,15 +1,63 @@
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
not fixed: 

![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/74ed6e51-1072-4510-9c6e-4fd653bfdf3d)

make sure lodash.debounce exists in the list of `dependencies` in package.json

add package.json to git and push it